### DISCUSSION TITLE
Option to add date to expenses

### DISCUSSION THREAD

**Discussion by Bugaddr** [danilkinkin/buckwheat#65] [Ideas]
Having an option to add date to expenses allows us to add old transactions to the list. Sometimes if I miss adding an entry then it becomes important to have such an option.

**Comment by danilkinkin** [comment]
Now it is possible to edit the date of an expense.

You can create a spent and immediately edit the date of the expense. Yes, it takes a little longer than if you could specify the desired date immediately upon creation, but it seems to me that this is an excessive overload of the interface
